nemo/collections/asr/models/ctc_models.py

```diff
@@ -35,6 +35,9 @@ class EncDecCTCModel(ASRModel):
     """Encoder decoder CTC-based models."""
 
     def __init__(self, cfg: DictConfig, trainer: Trainer = None):
+        if 'cls' not in cfg:
+            # This is for Jarvis service. Adding here for now to avoid effects of decorators
+            cfg['cls'] = 'nemo.collections.asr.models.EncDecCTCModel'
```

Comment:
If this is an immutable struct from DictConfig, you won't be able to add the "cls" key like this. You'd need to downcast to doct, add the key, then upcast to DictConfig again.